### PR TITLE
Priority card links/prop types

### PR DESCRIPTION
**Pull request by ryanmdoyle**
I added prop-types for type checking incoming props as well as reworked the PriorityCards so that they will show buttons to change the priority *only*  when on the correct page and made it so the entire button is a link to (eventually) the particular priority.  It was like this previously, but adding the buttons to rearrange required refactoring so the buttons changed the order without redirecting to another page.

There are also some files that kept setting of linting rules so I commented out some things and added alt tags to images.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "axios": "^0.19.0",
     "node-sass": "^4.12.0",
+    "prop-types": "^15.7.2",
```
Comment (kelfink) [inline package.json]: yeah, its about time we add these

File: src/components/ActionsPage.js [hunk 2]
```diff
@@ -22,22 +22,22 @@ const Header = styled.div`
...
-    window.open('http://twitter.com/share?url='+encodeURIComponent(url)+'&text='+encodeURIComponent(text), '', 'left=0,top=0,width=550,height=450,personalbar=0,toolbar=0,scrollbars=0,resizable=0');
+    window.open('http://twitter.com/share?url=' + encodeURIComponent(url) + '&text=' + encodeURIComponent(text), '', 'left=0,top=0,width=550,height=450,personalbar=0,toolbar=0,scrollbars=0,resizable=0');
   }
 
   render() {
     return (
       <ActionStyled>
         <Header>
-          <img src={'an image'} />
+          <img src={'an image'} alt={'some alt text'} />
```
Comment (kelfink) [inline src/components/ActionsPage.js]: We need something better here, obviously.  To be addressed soon

File: src/components/PriorityCard.js [hunk 3]
```diff
@@ -1,31 +1,58 @@
...
-      </Link>
-    </div>
-  );
+      </div>
+    );
+  }
 };
 
-PriorityCard.propTypes = {};
+PriorityCard.propTypes = { //needs a priorityId
```
Comment (kelfink) [inline src/components/PriorityCard.js]: yes!

File: src/components/ActionsPage.js (hunk 2 above)
Comment (ryanmdoyle) [reply src/components/ActionsPage.js]: Yes, currently there are no images associated with Actions.

**Pull request merged**